bullseye-slim/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM golang:1.18.2-bullseye AS builder
+
+LABEL maintainer="Skynet Labs <devs@skynetlabs.com>"
+
+WORKDIR /root
+
+ENV CGO_ENABLED 0
+ENV GOOS linux
+ENV GOARCH amd64
+
+ARG branch
+
+RUN git clone https://gitlab.com/SkynetLabs/skyd.git --single-branch --depth 1 --branch ${branch} && \
+    make build --directory skyd
+
+FROM debian:bullseye-slim
+
+LABEL maintainer="Skynet Labs <devs@skynetlabs.com>"
+
+# copy mime types
+COPY --from=builder /etc/mime.types /etc/mime.types
+
+# copy skyd and skyc binaries
+COPY --from=builder /root/skyd/bin/ /usr/bin
+
+ENV SIA_DATA_DIR /sia-data
+ENV SIAD_DATA_DIR /sia-data
+
+ENTRYPOINT [ "skyd" ]
+
+CMD [ "--disable-api-security", "--api-addr", ":9980", "--modules", "gtcwra" ]
```

Comment:
I don't think disabling api security by default is a good idea. A user should only do this if they know what they are doing. Not everyone might be running skyd behind a reverse proxy. Instead they might just run it locally like people use to with siad.

Reply:
I agree, initially I did not provide CMD at all and expected users to provide params manually, I can go back to that if we agree it's better to rely on skyd for defaults rather than overwrite them in image (I copied those defaults from docker-sia but that doesn't have to mean we need to do the same).

Reply:
Yeah I'd just use skyd's defaults for consistency instead of changing them to another set of defaults that we need to be aware of.

Reply:
Removed default command line arguments overrides in https://github.com/SkynetLabs/docker-skyd/pull/1/commits/9c720b2869e25dfd34c471b68667e499d4461515